File: C/EX31.c
#include<stdio.h>
#include<stdlib.h>

int main(int argc, char const *argv[])
{
    int a, b, c, d, i;

    printf("Serie de Bergamaschi");

    a = 1;
    b = 1;
    c = 1;
    d = 1;
    i = 0;

    for ( i = 0; i < 3; i++)
    {
        printf("\n%i", d);
    }
    
    d = 0;
    i = 0;

    while (i<20)
    {
        d = a + b + c;
        a = b;
        b = c;
        c = d;

        printf("\n%i", d);
        i++;
    }
    
    return 0;
}

File: Python/EX31.py
#série de BERGAMASCHI

print("Série de BERGAMASCHI")

#inicialização de variáveis
a = 1
b = 1
c = 1
d = 0
i = 0


#imprime os três primeiros números da série
while (i < 3):
    print(f"{c}")
    i+=1


i = 0


#apresenta os 20 números da serie
while (i < 20):
    d = a + b + c
    a = b
    b = c
    c = d
    print(f"{d}")
    i+=1

